exampleCourse/questions/template/number-input/multiple-answer-inputs/question.html

```diff
@@ -0,0 +1,13 @@
+<pl-question-panel>
+  <p>Compute the roots of the following polynomial:</p>
+  $$ {{params.polynomial}}$$
+  <p><strong>Enter the smaller root first, and the larger root second.</strong></p>
```

Comment:
This might also be a good question to demonstrate a custom grading function that allows the roots to be entered in any order, but that might also make the whole thing too complex?

Reply:
Yeah, I worried that that would blow up the complexity of this, which is exactly why I took the larger/smaller approach to guarantee that each input would have only one answer to grade. We should probably still add such a question, but I think that should be a different one.

Reply:
Yep, I'm on board with that.